.github/workflows/build-deb-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Test Deb Package
+
+# yamllint disable-line rule:truthy
```

Comment:
```suggestion
```

We don't run YAML lint, no need to exclude this line for something that does not run.